test/Test-EndToEnd.js

```diff
@@ -216,6 +216,184 @@ describe('EndToEnd', () => {
         });
     });
 
+    describe('FileStore with relativeLocation', () => {
+        let file_id;
+        let deferred_file_id;
+        before(() => {
+            server = new Server();
+            server.datastore = new FileStore({
+                path: STORE_PATH,
+                // configure the store to return relative path in Location Header
+                relativeLocation: true
+            });
+            agent = request.agent(server.listen());
+        });
+
+        after((done) => {
```

Comment:
This is unnecessary because it is already [declared for every test](https://github.com/tus/tus-node-server/blob/76ac93544a8c4fc0878dc00cfdd8c07f8dd09f49/test/Test-EndToEnd.js#L56-L67)